i18n/2017/categories/misc/ja.md

```diff
@@ -0,0 +1 @@
+TODO: Need translation.
```

Comment:
If @michaelrambeau is k with the category message, i would love some help for the translation @azukiwasher  ;)

Reply:
236072f82bebe646cede198c906cce6f06d0ed05